nengo_bones/templates/docs.sh.template

```diff
@@ -9,6 +9,7 @@
         "jupyter>=1.0.0" \
         "nbsphinx>=0.2.13" \
         "numpydoc>=0.6.0" \
+        "sphinx-notfound-page>=0.5.0" \
```

Comment:
Could/should we put this in `setup.py` for nengo-sphinx-theme? Perhaps in the "optional" section, then we can install with `[optional]` below.

Reply:
Ah yeah it should be there. I'd just put it in the main requirements, as I think we'd expect/want people to be using the 404 page extension in almost all cases (I can't think of any reason why you wouldn't), and then people don't need to remember the `[optional]` tag.